### PR TITLE
Added Puli support

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "require": {
         "php": ">=5.4",
         "php-http/httplug": "1.0.0-beta",
-        "php-http/discovery": "~0.5",
+        "php-http/discovery": "^0.6",
```
Comment (sagikazarmark) [inline composer.json]: Should explicitly be ^0.6.4. See the changelog for details.

File: puli.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "version": "1.0",
+    "bindings": {
```
Comment (sagikazarmark) [inline puli.json:4]: Please add `"name": "php-http/guzzle5-adapter",` after version.

**Comment by sagikazarmark** [comment]
Please add a more meaningful commit message than 'Fix 12' (but don't forget to add fixes 12 to it).

Looks good, thanks.

**Comment by Nyholm** [comment]
Thank you for your feedback. I've updated the PR

**Comment by Nyholm** [comment]
The build is breaking because of the integration tests requires `discovery 0.5`.

**Pull request closed without merging**